Collator.fs

```diff
@@ -202,6 +202,11 @@ module ParamDesugar =
                 | SymAtomic sym ->
                     SymAtomic (rewriteSymbolic sym)
                 | Havoc v -> Havoc (rewriteVar v)
+                | ACond (cond = c; trueBranch = t; falseBranch = f) ->
+                    ACond
+                        (rewriteExpression c,
+                         List.map rewriteAtomic t,
+                         Option.map (List.map rewriteAtomic) f)
```

Comment:
Adding conditionals to the AST rewrite.

Incidentally, this AST rewrite logic was put in so I could cheaply rewrite parameters of methods into local variables.  Eventually we should move that into the modeller and bin all of this rewrite crap.